client/components/editable-service-name/editable-service-name.less

```diff
@@ -14,15 +14,14 @@
 
         .btn-editable {
```

Comment:
custom rule ? :(

width + height fixed === bad because you can't see the overflow if needed (base font can be bigger than settled sometimes :/)

Uses min-width / min-height, line-height